consensus/consensus_test.go

```diff
@@ -700,7 +700,7 @@ func TestValidateBlockBody(t *testing.T) {
 		},
 		{
 			"ZeroGasTx", func(t *testing.T) {
-				txBuilder := new(tx.Builder).
+				txBuilder := new(tx.LegacyBuilder).
```

Comment:
is it a good place to add invalid tests for dyn tx ? like invalid max fee and max tip

Reply:
These will be included in the last piece with all the logic for fee calculation